src/sas/sasgui/guiframe/data_processor.py

```diff
@@ -18,7 +18,7 @@
 
 """
 import wx
-import numpy
+import numpy as np
 import math
 import re
```

Comment:
importing standard packages should be done before user packages. So the order here should be:

import re
import math
import wx
import numpy as np